ddcz/context_processors.py

```diff
@@ -17,3 +20,7 @@ def common_variables(request):
         "skin_logo_url": staticfiles_storage.url("skins/%s/img/logo.gif" % skin),
         "login_form": LoginForm(),
     }
+
+
+def skin_for_include(skin):
```

Comment:
This makes it available as a context processor, I'd either prefix with `_` or just embed it in the common vars